devito/finite_differences/derivative.py

```diff
@@ -134,30 +136,35 @@ def __new__(cls, expr, *dims, **kwargs):
         obj._fd_order = fd_orders
         obj._deriv_order = orders
         obj._side = kwargs.get("side", centered)
-        obj._stagger = kwargs.get("stagger", tuple([left]*len(obj._dims)))
         obj._transpose = kwargs.get("transpose", direct)
-        obj._eval_at = as_tuple(kwargs.get("eval_at"))
-
+        obj._subs = as_tuple(kwargs.get("subs"))
+        obj._x0 = kwargs.get('x0', {d: d for d in obj._dims})
         return obj
 
-    def _xreplace(self, eval_at):
+    def subs(self, *args, **kwargs):
+        """
+        Bypass sympy.Sub as Devito as its own lazy evaluation mecanism
```

Comment:
"sympy.subs"

as -> has

full stop at the end

Reply:
> as -> has

Is cockney not acceptable in Devito doc-strings?